project/pom.xml

```diff
@@ -1393,35 +1394,7 @@
             <configuration>
               <toolchains>
                 <jdk>
-                  <version>1.7</version>
-                  <vendor>oracle</vendor>
-                </jdk>
-              </toolchains>
-            </configuration>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <id>jenkins-jdk8</id>
-      <build>
-        <plugins>
-          <!-- make sure jclouds is built with the right JDK -->
-          <plugin>
-            <artifactId>maven-toolchains-plugin</artifactId>
-            <version>1.1</version>
-            <executions>
-              <execution>
-                <goals>
-                  <goal>toolchain</goal>
-                </goals>
-              </execution>
-            </executions>
-            <configuration>
-              <toolchains>
-                <jdk>
-                  <version>1.8</version>
-                  <vendor>openjdk</vendor>
+                  <version>${jdk.version}</version>
```

Comment:
@nacx Does removing the `oracle` vs. `openjdk` `<vendor>` attribute have any impact?

Reply:
No. We were not building against different JDK flavors, so it does not have an impact. If we want to do so, we can introduce the vendor again and revisit this. The property approach comes in handy for a clearer [jclouds-compat build](https://jclouds.ci.cloudbees.com/job/jclouds-compat/).